src/data/equiv/set.lean

```diff
@@ -432,39 +432,39 @@ of_left_inverse f (λ _, f_inv) (λ _, hf)
 
 /-- If `f : α → β` is an injective function, then domain `α` is equivalent to the range of `f`. -/
 @[simps apply]
-noncomputable def of_injective {α β} (f : α → β) (hf : injective f) : α ≃ set.range f :=
+noncomputable def of_injective {α β} {f : α → β} (hf : injective f) : α ≃ set.range f :=
```

Comment:
Rename to `_root_.function.injective.small`? Then you'll be able to write `hf.small`.